amr-wind/turbulence/RANS/KOmegaSST.H

```diff
@@ -85,6 +86,12 @@ protected:
     amrex::Real m_sigma_omega1{0.5};
     amrex::Real m_sigma_omega2{0.856};
     amrex::Real m_a1{0.31};
+
+    //! Buoyancy modified constants
+    bool m_include_buoyancy{false};
```

Comment:
I'd suggest enabling this by default only for multi-phase flows if it's an easy fix.